operator/v1/types_network.go

```diff
@@ -80,6 +80,11 @@ type NetworkSpec struct {
 	// If not specified, sensible defaults will be chosen by OpenShift directly.
 	// Not consumed by all network providers - currently only openshift-sdn.
 	KubeProxyConfig *ProxyConfig `json:"kubeProxyConfig,omitempty"`
+
+	// exportNetworkFlows enables and configures the export of network flows from OVS
```

Comment:
does this work with all network plugins?  kuryr for instance

Reply:
done